Wrappers/Python/test/test_algorithms.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
-#   (Collaborative Computational Project in Tomographic Imaging), with 
+#   This work is part of the Core Imaging Library (CIL) developed by CCPi
```

Comment:
No change in this file is functional. Is that true?

If so, please remove this file from this PR.

Reply:
Fixed.